miasm/jitter/vm_mngr.c

```diff
@@ -926,7 +926,7 @@ void hexdump(char* m, unsigned int l)
       if (!(i%0x10) && i){
       last = i;
       printf("    ");
-      for (j=-0x10;j<0;j++){
+      for (j=-0x10;(int)j<0;j++){
```

Comment:
Hey @qihuanpiaoliu !
Thank you for the fix.
For the last one, maybe we can declare an "int" (signed) `k` for example, and use it here to avoid the warning instead of casting j as unsigned. What do you thinl of this?